### ISSUE TITLE
NuGet package?

### ISSUE THREAD

**Issue by dazinator** [Kelindar/misakai-baker#3]
I'm looking to generate HTML documentation for a project, and am looking around for a static site generator that supports markdown for this purpose..

My ideal usage would be to add a NuGet package to my solution that contains misakai-baker (MB). Then I would call it during the build to generate the HTML documentation site.

Does this sound like a good potential fit for MB? Would you take PR's around  publishing MB as a NuGet package?

**Comment by kelindar** [comment]
Of course. The easiest way to do this would be a simple appveyor build modification adding the .nuget artifact.
